data/learning.yml

```
@@ -64,6 +64,13 @@ books:
       - link: /learn/best_practices/callbacks
       - link: /learn/best_practices/type_casting
 
+  - title: Python Fortran Rosetta Stone
+    description: This rosetta stone shows how to implement many common idioms in Python with NumPy and Fortran side by side.
```

```suggestion
    description: This "Rosetta Stone" shows how to implement many common idioms in Python with NumPy and Fortran side by side.
```
The lowercase feels kind of weird to me, but not sure of the best way.